### PR TITLE
Pipelines structure, attribute file, and folders and symlinks for pipelines

### PR DESCRIPTION
**Pull request by Preeticp**
This PR:
- Adds a commented out structure of docs for OpenShift pipeline in the OpenShift Pipeline bucket.
- Moves Pipelines bucket to be on the same level as Builds bucket instead of inside the Builds bucket.
- Adds an attribute file for OpenShift Pipelines
- Adds a tkn_cli bucket under the CLI Tools bucket
- Adds openshift_pipelines folder with symlinks
- Adds tkn_cli folder and symlinks

Note the modifications in the topic.yml are commented out as we are still DP and are likely to go TP by the end of this month.

### CODE REVIEW

**Comment by Preeticp** [comment]
@bmcelvee can you please take a look at this PR. It creates 2 buckets for Pipeline docs, one below the Builds bucket and the other one for Pipelines cli `tkn` in the CLI reference bucket. I am asking you cause you were designated as our buddy for the initial integration of Pipeline docs.
Initially, I thought the build failed because the sections were commented out, but now I think it is cause we do not have actual assemblies in place, is that correct?

File: _topic_map.yml [hunk 1]
```diff
@@ -1235,6 +1245,14 @@ Topics:
   Topics:
   - Name: Getting started with Helm on OpenShift Container Platform
     File: getting-started-with-helm-on-openshift-container-platform
+#- Name: tkn CLI
```
Comment (bmcelvee) [inline _topic_map.yml]: I think these should go in the CLI tools bucket.

Comment (bmcelvee) [reply _topic_map.yml]: Ignore that content - you've already got it there. Sorry!

File: cli_reference/tkn_cli/images [hunk 2]
```diff
@@ -0,0 +1 @@
+../images
```
Comment (bmcelvee) [inline cli_reference/tkn_cli/images]: The symlinks might be causing the build failure. Have you tried commenting them out? Or removing them and building locally?

Comment (Preeticp) [reply cli_reference/tkn_cli/images]: I will try that, thank you!

**Comment by Preeticp** [comment]
@bmcelvee request a review and merge, please. I checked with Vikram regarding the build failure, it was cause of the separator not being commented out. I have corrected that and have added the symlinks back.

**Comment by bmcelvee** [comment]
LGTM! Glad he was able to help figure it out! 

This applies to 4.4, correct? I just want to be sure before cherry picking it over.

**Comment by Preeticp** [comment]
Thank you Brandi, afaik, this will be an async release, as in, the docs will be published only once they go TP. They are aiming to do TP release before the month end, but it could be a bit delayed too. Hence I have commented out this section. I will confirm this again with the team and get back to you as soon as I have their opinion.

**Comment by Preeticp** [comment]
Hi @bmcelvee I just got confirmation from Siamak, since TP would be happening before the 4.4 GA date we would want to push the docs in 4.3

**Comment by bmcelvee** [comment]
Sounds good, thanks! I'm adding the 4.3 label and cherry picking to both 4.3 and 4.4.

**Pull request merged**